plugin_config.txt

```diff
@@ -959,6 +959,21 @@ my $VEP_PLUGIN_CONFIG = {
         # "/path/to/ClinPred_tabbed.tsv.gz"
       ]
     },
+    
+    # LOEUF
```

Comment:
Suggest adding comment only available for GRCh37
Is a param showing /path/to/file needed as per other plugins?